bin/dev

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+if gem list --no-installed --exact --silent foreman; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
```

Comment:
NIt: would it be worth checking if yarn is present and if not remind the user to install it?